subprojects/internal-integ-testing/src/main/groovy/org/gradle/integtests/fixtures/executer/UseJdk11OrLaterSampleModifier.java

```diff
@@ -0,0 +1,63 @@
+/*
+ * Copyright 2022 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.gradle.integtests.fixtures.executer;
+
+import com.google.common.collect.ImmutableMap;
+import org.gradle.api.JavaVersion;
+import org.gradle.exemplar.model.Sample;
+import org.gradle.exemplar.test.runner.SampleModifier;
+import org.gradle.integtests.fixtures.AvailableJavaHomes;
+import org.gradle.internal.jvm.Jvm;
+import org.gradle.util.internal.GUtil;
+
+import java.io.File;
+import java.util.Map;
+import java.util.Properties;
+
+/**
+ * Modifies runtime Java for samples that require Java11 or later if possible
+ */
+public class UseJdk11OrLaterSampleModifier implements SampleModifier {
```

Comment:
We should change these two locations instead: https://github.com/gradle/gradle/blob/2c6b35a6ee248ba9c04b3ee90a3c2974798ed8f0/subprojects/docs/build.gradle#L637-L640
https://github.com/gradle/gradle/blob/2c6b35a6ee248ba9c04b3ee90a3c2974798ed8f0/subprojects/docs/build.gradle#L612-L616

Reply:
Removed UseJdk11OrLaterSampleModifier and used exclussions